lib/response.js

```diff
@@ -137,7 +137,10 @@ module.exports = {
 
     // no content
     if (null == val) {
-      if (!statuses.empty[this.status]) this.status = 204;
+      if(!statuses.empty[this.status]) {
```

Comment:
Nit: space between if and parenthesis.

@jonathanong the reason this passes tests is because --fix is automatically run on travis, and so it'll fix it but the commit remains "invalid" according to eslintrc.yml.